Dockerfile

```
@@ -1,17 +1,19 @@
-# Default to EL7 builds
-ARG IMAGE_BASE_TAG=centos7
+# Default to EL8 builds
+ARG IMAGE_BASE=quay.io/centos/centos:stream8
 
-FROM centos:$IMAGE_BASE_TAG
+FROM $IMAGE_BASE
 
-# "ARG IMAGE_BASE_TAG" needs to be here again because the previous instance has gone out of scope.
-ARG IMAGE_BASE_TAG=centos7
+# "ARG IMAGE_BASE" needs to be here again because the previous instance has gone out of scope.
+ARG IMAGE_BASE=quay.io/centos/centos:stream8
 ARG BASE_YUM_REPO=testing
 ARG OSG_RELEASE=3.5
 
 LABEL maintainer OSG Software <help@opensciencegrid.org>
 
 RUN \
-    if  [[ $IMAGE_BASE_TAG == centos7 ]]; then \
+    # Attempt to grab the major version from the tag
+    DVER=$(egrep -o '[0-9][\.0-9]*$' <<< "$IMAGE_BASE" | cut -c1); \
```

```suggestion
    DVER=$(egrep -o '[0-9][\.0-9]*$' <<< "$IMAGE_BASE" | cut -d. -f1); \
```
otherwise centos 10 would break it. That being said - are you expecting a `.` in the tag at all? Or would
```
egrep -o '[0-9]+$'
```
be enough?

---

Good call

There are tags like `centos7.9.2009` so I figured we should try to protect against it